bin/git-reauthor

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+
+
+init_variables() {
+  COMMAND=${0#*-}
+
+  CONFIG=false
+  ALL=false
+  unset OLD_EMAIL
+  unset CORRECT_EMAIL
+  unset CORRECT_NAME
+  TYPE='full'
+}
+
+
+usage() {
+  cat << EOF
+usage: git ${COMMAND} [<options>]
+
+Options
+    -A, --all                     rewrite all commits and tags
+    -C, --config                  define correct values from user Git config
+    -e, --correct-email  <email>  define the correct email to set
+    -n, --correct-name   <name>   define the correct name to set
+    -o, --old-email      <email>  rewrite only commits and tags from old email
+    -t, --type           <id>     define the type of the rewrite
+                                  author, committer, full (default)
```

Comment:
After the rewrite of the descriptions, I came to agree to it :), in commit 4700320